app/(tabs)/toolbox.tsx

```diff
@@ -278,22 +282,66 @@ const processTools = (tools: Tool[], columns: number) => {
   return [...tools, ...placeholders];
 };
 
+function processBannerData(bannerData: LaunchScreenScreenResponse): BannerContent[] {
+  const banners: BannerContent[] = [];
+  for (const banner of bannerData) {
+    if (!banner.url) break;
+    if (banner.type === 2 && banner.href) {
+      // 网页跳转
+      banners.push({
+        image: { uri: banner.url },
+        text: banner.text ?? '',
+        type: BannerType.URL,
+        href: banner.href,
+      });
+    } else if (banner.type === 3 && banner.href) {
+      // 跳转 activity
+      banners.push({
+        image: { uri: banner.url },
+        text: banner.text ?? '',
+        type: BannerType.Activity,
+        href: banner.href,
+      });
+    } else {
+      // 纯图片
+      banners.push({
+        image: { uri: banner.url },
+        text: banner.text ?? '',
+        type: BannerType.NULL,
+      });
+    }
+  }
+  return banners;
+}
+
 // 自定义 Hook：管理横幅和工具数据
 const useToolsPageData = (columns: number) => {
+  const { data: bannerData } = useApiRequest(getApiV1LaunchScreenScreen, {
+    type: 2, // 1为开屏页，2为轮播图，3为生日当天的开屏页
```

Comment:
```suggestion
    type: 2, // 1为开屏页，2为轮播图
```

类型3已经弃用了，apifox那边没有删掉